lib/ansible/modules/windows/setup.ps1

```diff
@@ -130,6 +130,17 @@ foreach ($item in Get-ChildItem Env:) {
     $env_vars.Add($name, $value)
 }
 
+$domain_roles = @{
+    0 = "Stand-alone workstation"
+    1 = "Member workstation"
+    2 = "Stand-alone server"
+    3 = "Member server"
+    4 = "Backup domain controller"
+    5 = "Primary domain controller"
+}
+
+$domain_role = $domain_roles.Get_Item([Int32](Get-CimInstance Win32_ComputerSystem).DomainRole)
```

Comment:
Is it possible to reuse $win32_cs (L54) instead of calling Get-CimInstance Win32_ComputerSystem here again?